src/types.ts

```diff
@@ -0,0 +1,92 @@
+import { OpenmrsResource } from "@openmrs/esm-api";
+
```

Comment:
Are these types all relevant here or are these copied here from another similar page that dealt with patient lists and cohorts?